src/main/java/org/reaktivity/nukleus/socks/internal/stream/SocksServerFactory.java

```diff
@@ -676,19 +656,18 @@ private void doApplicationBegin(
             StringFW address,
             int port)
         {
-
             SocksBeginExFW socksBeginEx = socksBeginExRW.wrap(extBuffer, 0, extBuffer.capacity())
-                .typeId(socksTypeId)
-                .address(address)
-                .port(port)
-                .build();
+                                                        .typeId(socksTypeId)
+                                                        .address(t -> t.domainName(address.asString()))
```

Comment:
Same here, set address directly, avoid assumption of `domainName`.

Reply:
done